frontend/src/pages/auth/components/sign-in-form/sign-in-form.tsx

```diff
@@ -1,16 +1,63 @@
-import { Button } from '#libs/components/components.js';
+import { Button, Input, Link } from '#libs/components/components.js';
+import { AppRoute } from '#libs/enums/app-route.enum.js';
+import { useAppForm, useCallback } from '#libs/hooks/hooks.js';
+import {
+  type UserSignInRequestDto,
+  userSignInValidationSchema,
+} from '#packages/users/users.js';
+
+import styles from '../../styles.module.scss';
+import { DEFAULT_SIGN_IN_PAYLOAD } from './libs/constants.js';
 
 type Properties = {
-  onSubmit: () => void;
+  onSubmit: (payload: UserSignInRequestDto) => void;
 };
 
-const SignInForm: React.FC<Properties> = () => (
-  <>
-    <h1>Sign In</h1>
-    <form>
-      <Button label="Sign in" />
-    </form>
-  </>
-);
+const SignInForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
+  const { control, errors, handleSubmit } = useAppForm<UserSignInRequestDto>({
+    defaultValues: DEFAULT_SIGN_IN_PAYLOAD,
+    validationSchema: userSignInValidationSchema,
+  });
+
+  const handleFormSubmit = useCallback(
+    (event_: React.BaseSyntheticEvent): void => {
+      void handleSubmit(onSubmit)(event_);
+    },
+    [handleSubmit, onSubmit],
+  );
+
+  return (
+    <>
+      <form className={styles['authForm']} onSubmit={handleFormSubmit}>
```

Comment:
```suggestion
  
  return (
    <>
      <form className={styles['authForm']} onSubmit={handleSubmit(onSubmit)}>
```

Reply:
Yeah, already done) I copied that part from sign up template and thought it was needed there